core/base/supported.go

```diff
@@ -393,24 +390,6 @@ var centosSeries = map[SeriesName]seriesVersion{
 		Version:      "7",
 		Supported:    true,
 	},
-	Centos8: {
-		WorkloadType: OtherWorkloadType,
-		Version:      "8",
-		Supported:    true,
-	},
-	Centos9: {
-		WorkloadType: OtherWorkloadType,
-		Version:      "9",
-		Supported:    true,
-	},
```

Comment:
Centos9 is a rolling update, so can't be EOL, we need to put this one back in.

Reply:
Centos9 has been restored :cry: